js/src/carousel.js

```diff
@@ -160,7 +160,7 @@ class Carousel extends BaseComponent {
     }
     // End mod
 
-    // Boosted mod: if a play-pause button is present, set the button to play on mouseenter
+    // Boosted mod: if a play-pause button is present, set the button to play
     if (this._playPauseButton !== null && this._playPauseButton.classList.contains('pause')) {
       this._playPauseButton.classList.remove('pause')
       this._playPauseButton.classList.add('play')
```

Comment:
I had in mind but may be harder to maintain as you say
```js
this._playPauseButton.setAttribute('title', this._playPauseButton.getAttribute(SELECTOR_CAROUSEL_PLAY_TEXT) ? this._playPauseButton.getAttribute(SELECTOR_CAROUSEL_PLAY_TEXT) : SELECTOR_CAROUSEL_DEFAULT_PLAY_TEXT)
this._playPauseButton.querySelector('span.visually-hidden').innerHTML = this._playPauseButton.getAttribute(SELECTOR_CAROUSEL_PLAY_TEXT) ? this._playPauseButton.getAttribute(SELECTOR_CAROUSEL_PLAY_TEXT) : SELECTOR_CAROUSEL_DEFAULT_PLAY_TEXT
```

or even:
```js
this._playPauseButton.setAttribute('title', this._playPauseButton.getAttribute(SELECTOR_CAROUSEL_PLAY_TEXT) || SELECTOR_CAROUSEL_DEFAULT_PLAY_TEXT)
this._playPauseButton.querySelector('span.visually-hidden').innerHTML = this._playPauseButton.getAttribute(SELECTOR_CAROUSEL_PLAY_TEXT) || SELECTOR_CAROUSEL_DEFAULT_PLAY_TEXT
```

Reply:
If someone has a strong opinion on it, you can change it.
Otherwise I have the impression that we don't have much ternaries in the JS code (probably for maintainability reasons regarding the complexity of certain components like the carousel) so I don't mind having this "big" `if` on my side. Sounds readable and clear as is.